es6-test/unit/soql.js

```diff
@@ -35,7 +35,7 @@ describe('layer', function() {
     expect(t.name).to.equal('_1_foo');
   });
 
-  it.only('can launder multiple times', function() {
```

Comment:
Yay for specifying what tests you want to run in your source code.  x_x

It's convenient, but just seems like a recipe for disaster...